spec/models/draw_spec.rb

```diff
@@ -69,4 +69,22 @@
       expect(draw.open_suite_sizes).to eq([2])
     end
   end
+
+  describe '#available_suites' do
```

Comment:
Don't forget to remove these if you take out the methods (although RSpec should pick up on it either way)